apps/climatemappedafrica/src/components/DataVisualisationGuide/useStyles.js

```diff
@@ -0,0 +1,26 @@
+import makeStyles from "@mui/styles/makeStyles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
```

Comment:
Don't see any complicated CSS here; we should be able to switch to `sx` easily, no?